cranelift/filetests/src/zkasm_codegen.rs

```diff
@@ -188,6 +188,8 @@ fn fix_relocs(
             if let FinalizedRelocTarget::ExternalName(ExternalName::User(name)) = reloc.target {
                 let name = &params.user_named_funcs()[name];
                 if name.index == 0 {
+                    // Codegen line after migrating to new assert:
```

Comment:
```suggestion
                    // TODO(#1234): Codegen line after migrating to new assert:
```

Would be great to have a reference to something explaining what the plan for the new assert is and reference it here.